docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1061,4 +1061,23 @@ Default value: `/var/lib/clickhouse/access/`.
 
 -   [Access Control and Account Management](../../operations/access-rights.md#access-control)
 
+## user_directories {#user_directories}
+
+The path to the users config files. If `user_directories` is specified, the path from `users_config` won't be used.
```

Comment:
1. In the example I see two settings `users_xml` and `local_directory`. Do we have more? What does they mean?
     `user_directories` is not the settig, it is the section of configuration file. It contains settings.
2. Link to `users_config`.